client/blocks/login/two-factor-authentication/waiting-notification-approval.scss

```diff
@@ -1,3 +1,11 @@
+.two-factor-authentication__push-notification-approval {
+	width: 352px;
+	margin: 0 auto;
+	padding: 0;
+}
+
 .two-factor-authentication__info {
-	text-align: center;
+	text-align: left;
+	text-wrap: pretty;
```

Comment:
Same note on pretty, it's a lot of instances that will accumulate if we don't build this in at the root level.